docs/src/main/sphinx/functions/url.rst

```diff
@@ -40,7 +40,25 @@ such as ``:`` or ``?``.
 
 .. function:: url_extract_protocol(url) -> varchar
 
-    Returns the protocol from ``url``.
+    Returns the protocol from ``url``. e.g:
```

Comment:
Update to the our normal style for examples:

```
Returns the protocol from ``url``:

    SELECT url_extract_protocol('http://localhost:8080/req_path'); 
    -- http
    SELECT url_extract_protocol('https://127.0.0.1:8080/req_path');
    -- https

```

Reply:
Do need to keep the describe for each example? such as 'Extract ``https`` protocol::'

Reply:
@mosabua